### PR TITLE
extensions: Add EGL_ANDROID_telemetry_hint

### PR DESCRIPTION
**Pull request by pippy360**
Add a new EGL_TELEMETRY_HINT_ANDROID to help Android telemetry disguise between OpenGL contexts.

### CODE REVIEW

File: api/egl.xml [hunk 1]
```diff
@@ -1110,6 +1110,11 @@
         <enum value="0x8F74" name="EGL_CLIENT_PIXMAP_POINTER_HI"/>
     </enums>
 
+    <enums namespace="EGL" start="0x3570" end="0x3580" vendor="Google" comment="Reserved for Tom Murphy (Khronos bug ...)">
```
Comment (oddhack) [inline api/egl.xml:1113]: This block should move back to before the "<!-- Please remember" comment ca. line 1092.

Comment (oddhack) [inline api/egl.xml:1113]: Also the 'end="3580"' tag should be 'end="357F"'.

**Comment by pippy360** [comment]
Where do I get the bug id for the line:
 vendor="Google" comment="Reserved for Tom Murphy (Khronos bug ...)">
 ?

**Comment by oddhack** [comment]
> Where do I get the bug id for the line: vendor="Google" comment="Reserved for Tom Murphy (Khronos bug ...)"> ?

That dates back to when were using Bugzilla. I'd put in something like 'EGL-Registry pull #204' instead, these days.

**Comment by oddhack** [comment]
If this is something you're ready to publish in the near future, the PR as it stands is fine. If it will be in draft status for a while, we should pull out the reservations in egl.xml as a separate PR against main branch, so that they're actually reserved if someone else wants to allocate a new block.

**Comment by pippy360** [comment]
I would like to publish this as soon as possible

**Comment by oddhack** [comment]
@stonesthrow please set this to Approved to Merge if you're OK (and feel free to merge it, as well). This is non-objectionable as a vendor extension, so long as no other PR gets in there first with XML modifications that conflict.

**Comment by stonesthrow** [comment]
Status can change from "Draft", Dependencies, any versions of EGL with EGLConfigs. So left blank is OK.
Extension spec seems complete.

**Review by stonesthrow — APPROVED** [review]

File: index.php [hunk 2]
```diff
@@ -370,6 +370,8 @@
 </li>
 <li value=151> <a href="extensions/QNX/EGL_QNX_image_native_buffer.txt">EGL_QNX_image_native_buffer</a>
 </li>
+<li value=153> <a href="extensions/QNX/EGL_ANDROID_telemetry_hint.txt">EGL_ANDROID_telemetry_hint</a>
```
Comment (janharaldfredriksen-arm) [inline index.php:373]:
```suggestion
<li value=153> <a href="extensions/ANDROID/EGL_ANDROID_telemetry_hint.txt">EGL_ANDROID_telemetry_hint</a>
```

**Pull request merged**